### DISCUSSION TITLE
The Things Indoor Gateway | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1886] [Q&A]
# The Things Indoor Gateway | Seeed Studio Wiki

The Things Indoor Gateway

https://wiki.seeedstudio.com/The-Things-Indoor-Gateway/

**Comment by Pixeldieb** [comment]
Some Tips:

If you want to extend coverage it is possible to install a simple SMA Connector to the TTIG.

Happy to share pictures and/or a video of the process.
This enables external antennas like outdoor antennas to be installed to the gateway.
